app/src/main/kotlin/com/intive/picover/parties/view/PartyBottomSheet.kt

```diff
@@ -0,0 +1,93 @@
+package com.intive.picover.parties.view
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.datasource.LoremIpsum
+import androidx.compose.ui.unit.dp
+import com.intive.picover.R
+import com.intive.picover.common.text.PicoverOutlinedTextField
+import com.intive.picover.common.validator.ValidationStatus
+
+@Composable
+fun PartyBottomSheet(
+	title: String,
+	onTitleChange: (String) -> Unit,
+	titleValidationStatus: ValidationStatus,
+	description: String,
+	onDescriptionChange: (String) -> Unit,
+	descriptionValidationStatus: ValidationStatus,
+	buttonOnClick: () -> Unit,
+) {
+	Column(
+		Modifier.padding(16.dp),
+		horizontalAlignment = Alignment.CenterHorizontally,
+	) {
+		PicoverOutlinedTextField(
+			label = stringResource(R.string.PartyScreenAddPartyBottomSheetTitle),
+			modifier = Modifier
+				.fillMaxWidth()
+				.padding(bottom = 16.dp),
+			value = title,
+			onValueChange = onTitleChange,
+			isError = !titleValidationStatus.isValid(),
```

Comment:
In this implementation, the validation is checked only when a user changes the title (add or remove a character) (or I am wrong?), but shouldn't we also react when a user clicks on the field without changing it and go to the next field?

Reply:
@bartek977 any further comments?

Reply:
I agree